static/js/components/email/EmailCompositionDialog.js

```diff
@@ -86,6 +87,42 @@ export default class EmailCompositionDialog extends React.Component {
     }
   }
 
+  insertRecipientVariable = (variableName: string) => {
+
+    const { editorState } = this.state;
+    if(editorState !== null && editorState !== undefined) {
```

Comment:
I don't think this should ever be null since `editorState` is defined in the constructor